packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -300,24 +294,6 @@ export class SequenceInterval implements ISerializableInterval {
         return (endPos > bstart) && (startPos < bend);
```

Comment:
I'm surprised you didn't fix this too. bStart and bEnd.

Reply:
i have a bunch of things added to my user dictionary. i try not to be too strict about var names